docs/pages/TasksAndProcessors.md

```diff
@@ -1,57 +1,213 @@
 # Tasks and Processors
```

Comment:
I feel like I have asked this already in the past: Is there a reason why this page is not linked anywhere?

Reply:
I also think that the examples we have in the existing documentation pages are sufficient and also better, so I think we can remove this page.

Reply:
Yes, AFAIK we wanted to remove this page already in the past but linked to it from somewhere.

Reply:
Fine. I'll create a follow-up PR removing this page

Reply:
PR https://github.com/SAP/ui5-tooling/pull/742 covers this